docs/configuration-guides/platforms/packet.md

```diff
@@ -0,0 +1,263 @@
+| `enable_aggregation`                  | Enable the Kubernetes Aggregation Layer.                                                                                                                                      | true            | false    |
+| `networking`                          | CNI network plugin ("flannel" or "calico". )                                                                                                                                  | "calico"        | false    |
+| `network_mtu`                         | CNI interface MTU (applies only to calico).                                                                                                                                   | 1480            | false    |
+| `pod_cidr`                            | CIDR IPv4 range to assign Kubernetes pods.                                                                                                                                    | "10.2.0.0/16"   | false    |
+| `service_cidr`                        | CIDR IPv4 range to assign Kubernetes services.                                                                                                                                | "10.3.0.0/16"   | false    |
+| `cluster_domain_suffix`               | Cluster's DNS domain.                                                                                                                                                         | "cluster.local" | false    |
+| `enable_reporting`                    | Enables usage or analytics reporting to upstream. (applies to Calico only).                                                                                                   | false           | false    |
+| `reservation_ids`                     | Specify Packet hardware_reservation_id for instances.                                                                                                                         | -               | false    |
+| `reservation_ids_default`             | Default reservation ID for nodes not listed in the `reservation_ids`. The value`next-available` will choose any reservation that matches the pool's device type and facility. | ""              | false    |
+| `certs_validity_period_hours`         | Validity of all the certificates in hours.                                                                                                                                    | 8760            | false    |
+| `worker_pool`                         | Configuration block for worker pools. Can be more than one.                                                                                                                   | -               | true     |
+| `count`                               | Number of workers in the worker pool. Can be changed afterwards to add or delete workers.                                                                                     | 1               | true     |
```

Comment:
Should we have all these parameters prefixed with `worker_pool`? For example: `worker_pool.count` and so on... This would be consistent with what we do with the backend options.